### ISSUE TITLE
nvda saying microsoft word dialog twice:

### ISSUE THREAD

**Issue by wildcat11** [nvaccess/nvda#5498]
After creating a document in microsoft word and pressing alt plus f4 to close word 2010 nvda says microsoft word twice.

**Comment by ehollig** [comment]
I can confirm this in Word 2016, although I did not notice it until now.
### STR:
1. Open Microsoft Word
2. Add some text to the document
3. Exit by pressing Alt + F4
### Expected Behavior:
NVDA says "Microsoft Word Dialog" once.
### Actual Behaviour:
NVDA says "Microsoft Word Dialog" twice.
### Remarks:
Is this intentional or something that really needs to be fixed?

**Comment by jcsteh** [comment]
Annoying, but doesn't break functionality. P3.

**Comment by Adriani90** [comment]
@wildcat11, @ehollig I cannot reproduce this issue in MS Word 2010 and Word 365 with NVDA 2018.4.1 and with Alpha 16691,10590C87. Are you still having this issue?

**Comment by ehollig** [comment]
Nope. Can not reproduce anymore. Closing as worksforme
